### PR TITLE
New zero-downtime case examples

### PR DESCRIPTION
**Pull request by kadewu**
Add missing `removing index` case and as well cover foreign key case.

### CODE REVIEW

**Review by michal-macioszczyk — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: docs/developer/community/zero-downtime-migrations.mdx [hunk 1]
```diff
@@ -220,6 +220,52 @@ class Migration(migrations.Migration):
 
 Please note that line `atomic=False` is needed to proceed with concurrent index creation.
 
+### Removing the index from the database
+
+Similarly to adding the index to the database, removing the index also needs `concurrently` option.
```
Comment (2can) [inline docs/developer/community/zero-downtime-migrations.mdx:225]:
```suggestion
Similarly to adding the index to the database, removing the index also needs the `concurrently` option.
```

File: versioned_docs/version-3.x/developer/community/zero-downtime-migrations.mdx [hunk 2]
```diff
@@ -220,6 +220,52 @@ class Migration(migrations.Migration):
 
 Please note that line `atomic=False` is needed to proceed with concurrent index creation.
 
+### Removing the index from the database
+
+Similarly to adding the index to the database, removing the index also needs `concurrently` option.
```
Comment (2can) [inline versioned_docs/version-3.x/developer/community/zero-downtime-migrations.mdx:225]:
```suggestion
Similarly to adding the index to the database, removing the index also needs the `concurrently` option.
```

**Review by 2can — CHANGES_REQUESTED** [review]

**Comment by kadewu** [comment]
Alongside applying review suggestions, also added small note about nonatomic migrations that is needed in the concurrently build.

File: docs/developer/community/zero-downtime-migrations.mdx [hunk 3]
```diff
@@ -218,7 +218,54 @@ class Migration(migrations.Migration):
...
+        db_index=False,
+    )
+
+    class Meta:
+        indexes = BTreeIndex(fields=["related"], name="order_orderevent_related_id_idx")
+
+```
+
+And with this, you can follow [Adding the index to the database](#adding-the-index-to-the-database).
+Migration containing the `ForeighKey` creation should be separated from migration that creates an index concurrently.
```
Comment (2can) [inline docs/developer/community/zero-downtime-migrations.mdx:268]:
```suggestion
Migration containing the `ForeignKey` creation should be separated from migration that creates an index concurrently.
```

File: versioned_docs/version-3.x/developer/community/zero-downtime-migrations.mdx [hunk 4]
```diff
@@ -218,7 +218,54 @@ class Migration(migrations.Migration):
...
+        db_index=False,
+    )
+
+    class Meta:
+        indexes = BTreeIndex(fields=["related"], name="order_orderevent_related_id_idx")
+
+```
+
+And with this, you can follow [Adding the index to the database](#adding-the-index-to-the-database).
+Migration containing the `ForeighKey` creation should be separated from migration that creates an index concurrently.
```
Comment (2can) [inline versioned_docs/version-3.x/developer/community/zero-downtime-migrations.mdx:268]:
```suggestion
Migration containing the `ForeignKey` creation should be separated from migration that creates an index concurrently.
```

**Review by 2can — CHANGES_REQUESTED** [review]

**Review by 2can — APPROVED** [review]

**Pull request merged**